pkgs/servers/mail/sympa/default.nix

```diff
@@ -0,0 +1,116 @@
+{ stdenv, perlPackages, fetchurl, makeWrapper
+, dataDir ? "/var/lib/sympa"
+}:
+
+let
+  pname = "sympa";
+  version = "6.2.44";
+in
+stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
+
+  src = fetchurl {
```

Comment:
please use `fetchFromGitHub` instead

Reply:
Working on this. Is building from git generally preferred to building from release tarballs?